spec/date.html

```diff
@@ -65,16 +69,19 @@ <h1>Temporal.Date.from ( _item_ [ , _options_ ] )</h1>
         The following steps are taken:
       </p>
       <emu-alg>
-        1. Set _options_ to ? NormalizeOptionsObject(_options_).
         1. Let _overflow_ be ? ToTemporalOverflow(_options_).
         1. If Type(_item_) is Object, then
           1. Let _result_ be ? ToTemporalDateRecord(_item_).
         1. Else,
           1. Let _string_ be ? ToString(_item_).
           1. Let _result_ be ? ParseTemporalDateString(_string_).
         1. Let _constructor_ be the *this* value.
+        1. Let _calendar_ be _result_.[[Calendar]].
         1. Set _result_ to ? RegulateDate(_result_.[[Year]], _result_.[[Month]], _result_.[[Day]], _overflow_).
-        1. Return ? CreateTemporalDateFromStatic(_constructor_, _result_.[[Year]], _result_.[[Month]], _result_.[[Day]]).
+        1. If _calendar_ is *undefined*, set _calendar_ to ! GetDefaultCalendar().
+        1. Set _calendar_ to ? CalendarFrom(_calendar_).
+        1. Set _calendar_ to ? CalendarFrom(_calendar_).
```

Comment:
I think this is the wrong place; either move it into `ParseTemporalDateString` or into the `Else` block that calls it.